analyzer/tests/functional/analyze/test_analyze.py

```diff
@@ -705,6 +705,35 @@ def test_invalid_enabled_checker_name(self):
         errcode = process.returncode
         self.assertEqual(errcode, 0)
 
+    def test_disable_all_warnings(self):
+        """"""
+        build_json = os.path.join(self.test_workspace, "build.json")
+        analyze_cmd = [self._codechecker_cmd, "check", "-l", build_json,
+                       "--analyzers", "clang-tidy",
+                       "-d", "clang-diagnostic"]
```

Comment:
What happens if I disable all `clang-diagnostic` but I enable for example `clang-diagnostic-unused-variable`? Can we add a test case for this too?